modules/xfem/doc/content/modules/xfem/theory/theory.md

```diff
@@ -1,4 +1,127 @@
+\begin{equation}
+\boldsymbol{u}(\boldsymbol{x},t) = \underbrace{\sum_{I=1}^n N_I(\boldsymbol{x}) \boldsymbol{u}_I(t)}_{\text{Continuous}}
+                 + \underbrace{\sum_{J=1}^n N_J(\boldsymbol{x}) H(\boldsymbol{x})\boldsymbol{b}_J(t)}_{\text{Heaviside Enriched}}
+                 + \underbrace{\sum_{K=1}^n N_K(\boldsymbol{x}) \sum_{L=1}^4 F_L(\boldsymbol{x})c_K^L(t)}_{\text{Near-tip Enriched}},
+\end{equation}
+where $n$ is the number of nodes per finite element, $N_I$ are the standard finite element interpolation functions, $\boldsymbol{u}_I$ are the nodal displacements, $H$ is the Heaviside function, $\boldsymbol{b}_J$ are additional nodal degrees of freedom corresponding to Heaviside enrichment, $F_L$ are a set of enrichment functions that capture asymptotic near-tip fields, and $\boldsymbol{c}_K$ are additional nodal degrees of freedom corresponding to the near-tip enrichment. This approach requires an additional degree of freedom per solution variable for each node whose support is traversed by a crack for Heaviside enrichment, as well as 4 additional degrees of freedom per solution variable for the nodes in the vicinity of the crack tip to which near-tip enrichment is applied. The set of enriched nodes is typically a small subset of the nodes in the discretized solution because a small fraction of the solution domain is typically directly affected by cracking. As a result, this does not drastically increase the number of unknowns in the system of equations. However, this can be difficult to implement in existing FEM codes if they do not permit nodes to have arbitrary number of degrees of freedom.
+
+The phantom node method [!cite](Hansbo2004) was also proposed as a technique for modeling mesh-independent fracture.  In the phantom node method, elements intersected by a crack are deleted and replaced by two elements that occupy the same physical locations as the original elements. A portion of the domain of each of these new elements (partial elements) represents physical material. The combined physical portions of the partial elements cover the entire domain of the original element that was split. The nodes connected to the non-physical portions of those elements are known as phantom nodes. The solution field is interpolated on the partial elements using the standard finite element shape functions. When fields are integrated on partial elements, as is done to compute an element's contribution to the residual, the integral is only performed over the physical part of the element. Shortly after the phantom node technique was proposed, it was shown [!cite](Areias_Belytschko_2006) to give exactly the same enrichment as the originally proposed XFEM (without the near-tip enrichment functions), and add the same number of extra degrees of freedom to the system to represent discontinuities.
```

Comment:
This is all good here, but I didn't see a section discussing methods to enforce continuity between the fragmented elements. This stands on its own very well, but it would be nice to add a section discussing the need for solution continuity enforcement (due to us using phantom node method) and the ways this is accomplished (penalty method, Nitche's method).

Reply:
This theory section is basically copied and pasted from our paper. I might not call it theory actually, maybe implementation. Let us add the interface constraint later.